dao/src/main/java/cz.muni.fi.pa165.soccermanager/entity/Match.java

```diff
@@ -1,4 +1,113 @@
 package cz.muni.fi.pa165.soccermanager.entity;
 
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
 public class Match {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Temporal(TemporalType.DATE)
+    private Date date;
+    private String stadium;
+    @ManyToOne
+    private Team homeTeam;
+    @ManyToOne
+    private Team awayTeam;
+    private boolean finished;
+    private int homeTeamGoals;
+    private int awayTeamGoals;
+
```

Comment:
I think all of the attributes should be annotated as Column(nullable = false) because you propably dont want any of them to be null